apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableContainsInOrderAtLeast1ElementsOfAssertionSpec.kt

```diff
@@ -0,0 +1,37 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import org.spekframework.spek2.Spek
+import kotlin.reflect.KFunction3
+
+class IterableContainsInOrderAtLeast1ElementsOfAssertionSpec : Spek({
```

Comment:
please rename the class (and the file) to 
```suggestion
class IterableContainsInOrderOnlyElementsOfAssertionSpec : Spek({
```